src/pages/en/reference/api-reference.md

```diff
@@ -91,13 +91,13 @@ const data = await Astro.glob<CustomDataFile>('../data/**/*.js');
 
 ### `Astro.request`
 
-`Astro.request` is a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. It can be used to get the `url`, `headers`, `method`, and even body of the request. Use `new URL(Astro.request.url)` to get a URL object.
+`Astro.request` is a standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) object. It can be used to get the `url`, `headers`, `method`, and even body of the request. 
+
+See [`Astro.url`](#astrourl) for a full URL object of the current `Astro.request.url` value. This can be useful for accessing individual properties of the request URL, like pathname and origin.
 
 ```astro
----
-const url = new URL(Astro.request.url);
----
-<h1>Origin {url.origin}</h1>
+<h1>The current URL is: {Astro.request.url}</h1>
+<h1>The current URL pathname is: {Astro.url.pathname}</h1>
```

Comment:
What do you think about keeping the example here focused on `Astro.request` instead of mixing it up with `Astro.url`?

```suggestion
<p>Received a {Astro.request.method} request to {Astro.request.url}</p>
<p>The request had the following headers:</p>
<pre>{JSON.stringify(Object.fromEntries(Astro.request.headers), null, 2)}</pre>
```

Reply:
I like it! Updated